docker-compose.yml

```
@@ -30,9 +42,26 @@ services:
     networks:
       - app_network
 
+  backend-price-stream:
+    build: ./backend
+    env_file: .env
+    restart: unless-stopped
+    depends_on:
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+    volumes:
+      - ./backend:/backend:delegated
+      - /backend/.venv
+    networks:
+      - app_network
+    command: uv run manage.py mock_stream_prices
```

https://channels.readthedocs.io/en/latest/topics/worker.html#receiving-and-consumers

```suggestion
    command: uv run manage.py runworker mock_stream_prices
```

Musisz dodac chyba do `INSTALLED_APPS` channels, zrobic ze "streamow" consumery i zrobic ChannelNameRouter w `asgi.py`